### PR TITLE
Update migration notes with Play WS

### PR DESCRIPTION
**Pull request by wsargent**

### CODE REVIEW

File: documentation/manual/releases/Migration24.md [hunk 1]
```diff
@@ -332,9 +332,18 @@ If you use the Java API, the [`F.Promise`](api/java/play/libs/F.Promise.html) cl
...
 
-Play has upgraded from AsyncHttpClient 1.8 to 1.9, which includes a number of breaking changes if using or configuring that library directly.  Please see the [AsyncHttpClient Migration Guide](https://github.com/AsyncHttpClient/async-http-client/blob/master/MIGRATION.md) for more details.
+WS has upgraded from AsyncHttpClient 1.8 to 1.9, which includes a number of breaking changes if using or configuring that library directly.  Please see the [AsyncHttpClient Migration Guide](https://github.com/AsyncHttpClient/async-http-client/blob/master/MIGRATION.md) for more details.  The upgrade to AsyncHttpClient 1.9.x enables Server Name Indication (SNI) in HTTPS -- this solves a number of problems with HTTPS based CDNs such as Cloudflare which depend heavily on SNI.
+
+Configuration settings for WS have changed:
+
+* "ws.acceptAnyCertificate" has been moved under the loose settings as "play.ws.loose.acceptAnyCertificate" to better indicate the insecure nature of blindly accepting any X.509 certificate.
+* A number of AsyncHttpClient-specific options previously defined as "ws.ning" have been renamed to conform with AHC 1.9 settings.
```
Comment (richdougherty) [inline documentation/manual/releases/Migration24.md]: Are the renamed settings documented anywhere?

File: documentation/manual/releases/Migration24.md [hunk 2]
```diff
@@ -332,9 +332,18 @@ If you use the Java API, the [`F.Promise`](api/java/play/libs/F.Promise.html) cl
...
+`WSRequestHolder` has been renamed to `WSRequest` in [Scala](api/scala/index.html#play.api.libs.ws.WSRequest) and [Java](api/java/play/libs/ws/WSRequest.html).  The previous `WSRequest` class has been removed out as it was only used internally to WS for OAuth functionality.
 
-Play has upgraded from AsyncHttpClient 1.8 to 1.9, which includes a number of breaking changes if using or configuring that library directly.  Please see the [AsyncHttpClient Migration Guide](https://github.com/AsyncHttpClient/async-http-client/blob/master/MIGRATION.md) for more details.
+WS has upgraded from AsyncHttpClient 1.8 to 1.9, which includes a number of breaking changes if using or configuring that library directly.  Please see the [AsyncHttpClient Migration Guide](https://github.com/AsyncHttpClient/async-http-client/blob/master/MIGRATION.md) for more details.  The upgrade to AsyncHttpClient 1.9.x enables Server Name Indication (SNI) in HTTPS -- this solves a number of problems with HTTPS based CDNs such as Cloudflare which depend heavily on SNI.
```
Comment (richdougherty) [inline documentation/manual/releases/Migration24.md]: You could mention the SNI support under `Highlights24.md` if you like.

File: documentation/manual/releases/Migration24.md (hunk 1 above)
Comment (marcospereira) [inline documentation/manual/releases/Migration24.md]: Here? https://github.com/playframework/playframework/blob/master/documentation/manual/working/scalaGuide/main/ws/ScalaWS.md#configuring-asyncclientconfig

But... why there is **`ws.ning`**.`webSocketIdleTimeout`?

Comment (wsargent) [inline documentation/manual/releases/Migration24.md]: Good catch.  Looks like https://github.com/playframework/playframework/commit/122eedda94a1f5249a95d3f8e701b81a93becf61 changed the docs but NingConfigSpec.scala confirms that it is now "play.ws.ning.webSocketIdleTimeouts"

File: documentation/manual/releases/Migration24.md [hunk 3]
```diff
@@ -332,9 +332,32 @@ If you use the Java API, the [`F.Promise`](api/java/play/libs/F.Promise.html) cl
...
+
+Configuration settings for WS have changed.
+
+* "ws.acceptAnyCertificate" has been moved under the loose settings as `play.ws.loose.acceptAnyCertificate` to better indicate the insecure nature of blindly accepting any X.509 certificate without validation.
+
+Because of the AsyncHttpClient 1.9.x upgrade, several settings no longer have the same names that they did previously in the 1.8.x version AsyncHttpClientConfigBuilder.  To reduce confusion, here is the map from WS settings to 1.9.x AsyncHttpClientConfigBuilder:
+
+* `play.ws.ning.connectionTimeout` -> <a href="http://static.javadoc.io/com.ning/async-http-client/1.9.22/com/ning/http/client/AsyncHttpClientConfig.Builder.html#setConnectTimeout(int)">setConnectTimeout</a>
+* `play.ws.ning.idleTimeout` -> <a href="http://static.javadoc.io/com.ning/async-http-client/1.9.22/com/ning/http/client/AsyncHttpClientConfig.Builder.html#setReadTimeout(int)">setReadTimeout</a>
+* `play.ws.ning.followRedirects` -> <a href="http://static.javadoc.io/com.ning/async-http-client/1.9.22/com/ning/http/client/AsyncHttpClientConfig.Builder.html#setFollowRedirect(boolean)">setFollowRedirect</a>
```
Comment (richdougherty) [inline documentation/manual/releases/Migration24.md]: Just checking this config option is meant to be singular, not plural like the AHC method.

Comment (richdougherty) [inline documentation/manual/releases/Migration24.md]: I see that quite a few settings are named differently from the AHC properties, so I assume it's intentional.

**Comment by richdougherty** [comment]
LGTM :+1:

**Pull request merged**